scripts/generate_test_certs.sh

```diff
@@ -78,4 +83,8 @@ certstrap --depot-path ${depot_path} sign metricserver_client --CA autoscaler-ca
 
 # metricsforwarder certificate for loggregator_agent
 certstrap --depot-path ${depot_path} request-cert --passphrase '' --domain metron
-certstrap --depot-path ${depot_path} sign metron --CA loggregator-ca --years "20"
+certstrap --depot-path ${depot_path} sign metron --CA local-ca --years "20"
+
+# mTLS client certificate for local testing
+certstrap --depot-path ${depot_path} request-cert --passphrase '' --domain local_client --ou "app-id:an-app-id"
```

Comment:
It might make sense to test with more realistic certs, containing additional OUs:
> The Organizational Unit property in the certificate’s Subject Distinguished Name contains the values organization:ORG-GUID, space:SPACE-GUID, and app:APP-GUID. The ORG-GUID, SPACE-GUID, and APP-GUID are set to the GUIDs for the organization, space, and app as assigned by Cloud Controller.
https://docs.cloudfoundry.org/devguide/deploy-apps/instance-identity.html

Reply:
currently handled by openssl as certstrap does not allow multiple orgs